src/site/stages/build/plugins/output-html.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-param-reassign */
+
+const outputHtml = files => {
+  for (const fileName of Object.keys(files)) {
+    const file = files[fileName];
+    if (file.parsedContent && file.modified) {
+      file.contents = new Buffer(file.parsedContent.html());
+    }
+  }
+};
+
+module.exports = outputHtml;
```

Comment:
Instead of `output-html`, maybe something like `finalize-html`? Or something verbose like `replace-file-contents-with-dom` 🤷‍♂